rpa-builder/6.7/modules/ROOT/pages/toolbox-mail-operations-mail-session-oauth.adoc

```diff
@@ -4,10 +4,16 @@ The *Mail Session (OAuth)* action step is an inclusive element for email operati
 
 == Before You Begin
 
-Before you configure this action step, ensure you have: 
+Depending on the mail provider you use, ensure you have the following:
 
-* An Azure account, if you are creating an application for Outlook
-* A Google account, if you are creating an application for Gmail
+* *Outlook*
+** An Azure account
+** SMTP AUTH enabled for your Microsoft organization or the specific mail when connecting with an Azure environment via SMTP
```

Comment:
**Question**: Shouldn't this say "... or the specific mail**box** when connecting ..."?